src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/SpecValidationUtility.cs

```diff
@@ -315,7 +315,7 @@ private static void ValidateProjectSpecOther(PackageSpec spec, IEnumerable<strin
                 var message = string.Format(
                     CultureInfo.CurrentCulture,
                     Strings.PropertyNotAllowed,
-                    nameof(spec.Dependencies));
+                    "dependencies");
```

Comment:
nit?  Technically this changes the casing, right?

```suggestion
                    "Dependencies");
```

Reply:
yes, but honestly, it's not super important since this is now a file exclusively generated or read by NuGet, so it's now internal.